pynq/lib/__init__.py

```diff
@@ -41,9 +41,6 @@
 from .rgbled import RGBLED
 from .switch import Switch
 from .button import Button
-from .pmod import *
-from .arduino import *
-from .intf import *
```

Comment:
This will be a problem when you have
```python
from pynq.lib import ARDUINO
```
Where `ARDUINO` is defined both in `arduino` and `intf` subpackages. If we want to avoid that, either we need to use the same definition, or we explicitly define an input dictionary... Our notebooks will have to be changed in the latter case.